vips/vip-372/bscmainnet.ts

```diff
@@ -0,0 +1,186 @@
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: ARBITRUM_CHAIN_ID,
+      },
+      {
+        target: OPBNBMAINNET_XVS_BRIDGE_ADMIN,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: OPBNBMAINNET_CHAIN_ID,
+      },
+      {
+        target: ETHEREUM_XVS_BRIDGE_ADMIN,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: ETHEREUM_CHAIN_ID,
+      },
+      {
+        target: arbitrumone.CHAINLINK_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ONE_BOUND_VALIDATOR,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: arbitrumone.REDSTONE_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: arbitrumone.RESILIENT_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: opbnbmainnet.BINANCE_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+      {
+        target: opbnbmainnet.RESILIENT_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+      {
+        target: OPBNBMAINNET_BOUND_VALIDATOR,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+
+      {
+        target: ethereum.CHAINLINK_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
+
+      {
+        target: ethereum.REDSTONE_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ethereum.RESILIENT_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_BOUND_VALIDATOR,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_sFrxETH_ORACLE,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: arbitrumone.VTREASURY,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: opbnbmainnet.VTREASURY,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+      {
+        target: ethereum.VTREASURY,
+        signature: "acceptOwnership()",
+        params: [],
+        dstChainId: LzChainId.ethereum,
+      },
```

Comment:
TODO: add commands to grant and revoke permissions using the ACMCommandsAggregator contract